scss/standalone/patterns_icons-additional.scss

```diff
@@ -107,3 +107,52 @@
 @include vf-p-icon-user-group;
 @include vf-p-icon-video-play;
 @include vf-p-icon-warning-grey;
+
+@include vf-p-icon-archive;
+@include vf-p-icon-arrow-bottom-left;
```

Comment:
Nitpick here as this is purely an internal concern: perhaps these blocks of additional icon imports could also be merged and alphabetized

Reply:
I guess it should be fine to merge and alphabetise the list here as it's not exposed to users in docs and the specific historical order doesn't matter that much.

Reply:
Fixed